vcd/resource_vcd_catalog_item.go

```diff
@@ -236,3 +246,50 @@ func createOrUpdateCatalogItemMetadata(d *schema.ResourceData, meta interface{})
 	}
 	return nil
 }
+
+// Imports a CatalogItem into Terraform state
+// This function task is to get the data from vCD and fill the resource data container
+// Expects the d.ID() to be a path to the resource made of Org name + dot + Catalog name
```

Comment:
"Org name + dot + Catalog name" is not correct here. Overall, the way you wrote it down below is much easier to read than this ""Org name + dot + Catalog name". I mean, this form is a lot better:
`// Example import path (id): org_name.catalog_name.catalog_item_name`

Please switch to this cleaner/easier to scan form in all the import function comments of this PR as opposed the " + dot + ".

Reply:
It was the same convention that was approved for Org User importing.
I will switch to this one

Reply:
done